scss/templates/_featured.scss

```diff
@@ -0,0 +1,8 @@
+.rev-Featured {
+  p {
+    font-size: $font-size-small;
```

Comment:
is this something that should be a featured var in case there's an instance where they don't want to use the small font size? `$featured-font-size`?